zulip_bots/zulip_bots/bots/github_detail/github_detail.py

```diff
@@ -74,9 +74,16 @@ def handle_message(self, message: Dict[str, str], bot_handler: Any) -> None:
             return
 
         # Capture owner, repo, id
-        issue_prs = re.finditer(
-            self.HANDLE_MESSAGE_REGEX, message['content'])
+        issue_prs = list(re.finditer(
+            self.HANDLE_MESSAGE_REGEX, message['content']))
         bot_messages = []
+        if len(issue_prs) > 5:
+            bot_messages.append('Please ask for <=5 links in any one request')
```

Comment:
The `append` and `join` seem unnecessary here when we know we're just replying with a simple error message.  Can you simplify this?

Reply:
I have made the changes @showell

Reply:
Did you push the changes?

Reply:
My bad, give it a look now.